run-cleanup-in-container.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+IMAGE_NAME="usercont/bots-deployment"
+
+if [ -z "${DEPLOYMENT}" ]; then
+    echo "DEPLOYMENT has to be defined (to one of prod/stage/dev/test)"
+    exit 1
+fi
+
+if [ "${DEPLOYMENT}" == "prod" ]; then
+    echo "Hey buddy, I can't let you wipe out production deployment, sorry :-("
+    exit $EKILLEMALL
```

Comment:
lol

Reply:
For your own good guys :-) I know @dhodovsk already hit it once